src/UtilsUI.vala

```diff
@@ -56,29 +56,52 @@ public class FeedReader.UtilsUI : GLib.Object {
 	public static string buildArticle(string html, string title, string url, string? author, string date, string feedID)
 	{
 		var article = new GLib.StringBuilder();
+    string theme = Settings.general().get_string("article-theme");
+
 		string author_date = "";
 		if(author != null)
 			author_date +=  _("posted by: %s, ").printf(author);
 
 		author_date += date;
 
-		try
-		{
-			uint8[] contents;
-			var file = File.new_for_uri("resource:///org/gnome/FeedReader/ArticleView/article.html");
-			file.load_contents(null, out contents, null);
-			article.assign((string)contents);
-		}
-		catch(GLib.Error e)
-		{
-			Logger.error("Utils.buildArticle: %s".printf(e.message));
-		}
+    string template = "";
+    string css = "";
+    try
+    {
+        if (ArticleTheme.isExists(theme) == false) {
+          // only when none of the themes are found!
+            uint8[] templateContents;
+            var htmlFile = File.new_for_uri("resource:///org/gnome/FeedReader/ArticleView/default/article.html");
+            htmlFile.load_contents(null, out templateContents, null);
+            template = (string)templateContents;
+            uint8[] cssContents;
+            var cssFile = File.new_for_uri("resource:///org/gnome/FeedReader/ArticleView/default/style.css");
+            cssFile.load_contents(null, out cssContents, null);
+            css = (string)cssContents;
+        } else {
+
+          GLib.FileUtils.get_contents(theme + "/article.html", out template);
+          GLib.FileUtils.get_contents(theme + "/style.css", out css);
+      }
+    }
+    catch(GLib.Error e)
+    {
+      Logger.error("Utils.buildArticle: load article theme: " + e.message);
+    }
+
+    article.assign(template);
+
 
 		string html_id = "$HTML";
 		int html_pos = article.str.index_of(html_id);
 		article.erase(html_pos, html_id.length);
 		article.insert(html_pos, html);
 
+    string css_id = "$CSS";
+    int css_pos = article.str.index_of(css_id);
+    article.erase(css_pos, css_id.length);
+    article.insert(css_pos, css);
```

Comment:
It was used before by @jangernert, i will see what can i do. But i was going to remove including the CSS file here as it's not needed. it's even better to just load the html file, this way third-party themes could contain images, fonts that we shouldn't load by ourselves (vala code)